src/pages/LiquidityPool/LiquidityPool.tsx

```diff
@@ -294,6 +295,7 @@ const LiquidityPool: React.FC = () => {
                 const txResult = await tx.wait();
 
                 if (txResult && txResult.events) {
+                    plausible.trackEvent('deposit-lp');
```

Comment:
Can you make a separate files for this event and list them? For example in utils folder make events.ts file and list them as constants there